### PR TITLE
Allow required fields to be highlighted instead of optional fields

### PR DESCRIPTION
**Pull request by kbayliss**
Addresses https://github.com/torchbox/tbxforms/issues/68

Adds a new `TBXFORMS_HIGHLIGHT_REQUIRED_FIELDS` setting which will mark required fields with `*` instead of marking optional fields with `(optional)`.

Also:
* adds some missing tests
* allows developers to more easily target the required/optional `<span>` elements with two new classes: `.tbxforms-field_marker--required` and `.tbxforms-field_marker--optional`, allowing each variant to be independently styled.

Example when `TBXFORMS_HIGHLIGHT_REQUIRED_FIELDS=False` or not set (default behaviour):

Highlight optional

Example when `TBXFORMS_HIGHLIGHT_REQUIRED_FIELDS=True`:

Highlight required

### CODE REVIEW

**Comment by kbayliss** [comment]
@zerolab Hola! Does this address your needs?

File: README.md [hunk 1]
```diff
@@ -336,6 +336,15 @@ Layout(
...
 
+### Highlight required fields instead of optional ones
+
+If `TBXFORMS_HIGHLIGHT_REQUIRED_FIELDS=False` (or unset), optional fields will
+have "(optional)" appended to their labels. This is the default behaviour and
+recommended by GDS.
+
+If `TBXFORMS_HIGHLIGHT_REQUIRED_FIELDS=True`, required fields will have an
+asterisk appended to their labels and optional fields will not be highlighted.
+
```
Comment (zerolab) [inline README.md:347]: suggestion: highlight the marker classes (`tbxforms-field_marker--required`, `tbxforms-field_marker--optional`) should devs want to change their styling

**Review by zerolab — APPROVED** [review]
One non-blocking suggestion. This would do the trick rather nicely

File: README.md (hunk 1 above)
Comment (kbayliss) [reply README.md:347]: Good shout, done in https://github.com/torchbox/tbxforms/pull/78/commits/4fa218b6db64b90551397538a5f5f0b2a4584c7a

**Pull request merged**